tests/nfs/nfs_operations.py

```diff
@@ -1315,3 +1315,113 @@ def nfs_log_parser(client, nfs_node, nfs_name, expect_list=None):
                 log.info(
                     "Since we are collecting logs, ignoring the exception will not fail the test"
                 )
+    return 0
+
+
+def verify_ops_control_settings(client, cluster_name, export_path=None):
```

Comment:
Remove the changes from NFS operation file.
We can move the qos related changes to 1 common file and call the function from there like how we did for QoS BW 
https://github.com/red-hat-storage/cephci/blob/main/tests/nfs/test_nfs_qos_on_cluster_level_enablement.py